src/main/scala/com/softwaremill/realworld/users/User.scala

```diff
@@ -0,0 +1,33 @@
+package com.softwaremill.realworld.users
+
+import java.time.Instant
+
+case class User(
+    user: UserData
```

Comment:
because this is how [specification](https://realworld-docs.netlify.app/docs/specs/backend-specs/api-response-format#users-for-authentication) looks like:
{
  "user": {
    "email": "jake@jake.jake",
    "token": "jwt.token.here",
    "username": "jake",
    "bio": "I work at statefarm",
    "image": null
  }
}